modules/silauth/public/status.php

```diff
@@ -28,5 +28,5 @@
         $t->getMessage(),
         $t->getCode()
     );
-    \http_response_code(500);
+    http_response_code(500);
```

Comment:
I would have expected a more Yii'ish way of doing the same thing, not something so direct.

Reply:
Yeah, it is kinda stark, isn't it?